test/dynamo/test_functions.py

```diff
@@ -470,7 +470,11 @@ def test_get_default_dtype(x):
     def test_get_autocast_gpu_dtype(x):
         dtype = torch.get_autocast_gpu_dtype()
         return x.type(dtype)
-
+      
+    @make_test
+    def test_generated_custom_object_types(x):
+        return type(x)
```

Comment:
x is a tensor, this seems pretty unlikely to trigger the error case...

Reply:
test edited. @ezyang .

Reply:
Could you review it? @ezyang .